partner_compassion/models/parter_segment_affinity.py

```diff
@@ -0,0 +1,135 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Jonathan Guerne <jonathan.guerne@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, fields, models
+from math import exp
+
+
+class ResPartnerSegmentAffinity(models.Model):
+    _name = "res.partner.segment.affinity"
+
```

Comment:
- Add a description attribute in new models to give a human readable name of the model.
- Add default order attribute to ensure records are read in a given order
```suggestion
_description = "Partner segment affinity"
_order = "partner_id desc, affinity desc"

```